examples/03-Adding-Datasets/adding-camels-cl.ipynb

```diff
@@ -6,19 +6,19 @@
    "source": [
     "# Adding a New Dataset\n",
     "\n",
-    "There exist two different options to use a different dataset within the `neuralHydrology` library.\n",
+    "There exist two different options to use a different dataset within the `neuralhydrology` library.\n",
```

Comment:
```suggestion
    "There exist two different options to use a different dataset within the `neuralHydrology` library.\n",
```

This was on purpose, since we refer to the library as neuralHydrology with upper case "H". Should be like this everywhere, where we refer to the package name.

Reply:
Thanks, that makes sense